guide/english/ruby/common-array-methods/index.md

```diff
@@ -160,22 +162,20 @@ array.flatten
 ```
 
 #### .join
-The .join method returns a string of all the elements of the array separated by a separator parameter. If the separator parameter is nil, the method uses an empty string as a separator between strings.
+The .join method returns a string of all the elements of the array separated by a separator parameter. If the separator parameter is nil, the method uses an empty string as a separator between strings:
 
 ``` ruby
 array.join
-=> "1234"
+=> "01234"
 array.join("*")
-=> "1*2*3*4"
+=> "0*1*2*3*4"
```

Comment:
Here without declaration, the compiler won't know what values will be stored inside the `array` so add a line declaring and defining the value of `array`, something like this:
```ruby
array = [0, 1, 2, 3, 4]
```